src/neoxp/Commands/ExecuteCommand.cs

```diff
@@ -0,0 +1,250 @@
+// Copyright (C) 2015-2023 The Neo Project.
+//
+// ExecuteCommand.cs file belongs to neo-express project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Akka.Util.Internal;
+using McMaster.Extensions.CommandLineUtils;
+using Neo;
+using Neo.BlockchainToolkit;
+using Neo.IO;
+using Neo.Network.P2P.Payloads;
+using Neo.SmartContract;
+using Neo.VM;
+using System.ComponentModel.DataAnnotations;
+using System.Numerics;
+using System.Text;
+
+namespace NeoExpress.Commands
+{
+    [Command(Name = "execute", Description = "Invoke a custom script, the input text will be converted to script with a priority: hex, base64, file path.")]
+    class ExecuteCommand
+    {
+
+        readonly ExpressChainManagerFactory chainManagerFactory;
+        readonly TransactionExecutorFactory txExecutorFactory;
+
+        public ExecuteCommand(ExpressChainManagerFactory chainManagerFactory, TransactionExecutorFactory txExecutorFactory)
+        {
+            this.chainManagerFactory = chainManagerFactory;
+            this.txExecutorFactory = txExecutorFactory;
+        }
+
+        [Argument(0, Description = "A neo-vm script(hex,base64) or a script file path")]
+        [Required]
+        internal string InputText { get; set; } = string.Empty;
+
+        [Option(Description = "Account to pay invocation GAS fee")]
+        internal string Account { get; init; } = string.Empty;
+
+        [Option(Description = "Witness Scope to use for transaction signer (Default: CalledByEntry)")]
```

Comment:
`Witness Scope for transaction signer(s) (Allowed: None, CalledByEntry, Global)`